groups/groups.yaml

```diff
@@ -846,6 +849,7 @@ groups:
     settings:
       ReconcileMembers: "true"
     members:
+      - k8s-infra-release-editors@kubernetes.io
```

Comment:
Fine with this; maybe `k8s-infra-staging-cip-test` should be moved to sig-release/groups.yaml?

Reply:
Done!